tools/dm/benchmark.md

```diff
@@ -0,0 +1,130 @@
+---
+title: DM Benchmark Report
+summary: DM benchmark report
+category: tools
+---
+
+# DM Benchmark Report
+
```

Comment:
Adding a brief summary sentence under the 1st level heading would be better:) 

"This DM benchmark report describes the test purpose, environment, scenario, and result."